cartridges/int_cloudinary/cartridge/scripts/model/cloudinaryModel.js

```diff
@@ -355,7 +356,17 @@ var geContentVideoByName = function (videoName) {
 
                 var isvideoPlayerEnabled = cloudinaryHelper.isVideoPlayerEnabledForContentLibrary();
                 if (isvideoPlayerEnabled) {
-                    var widgetOptions = cloudinaryHelper.getContentVideoPlayerOptions();
+                    var widgetOptions = null;
+                    if(!empty(configs)) {
+                        try {
+                            widgetOptions = JSON.parse(configs);
+                        } catch (ex) {
+                            logger.error('Error occurred while parsing the video player configs from content asset: error: {0}, fileName: {1}, lineNumber: {2}', ex, ex.fileName, ex.lineNumber);
+                            widgetOptions = cloudinaryHelper.getContentVideoPlayerOptions();
```

Comment:
@asad-rafter I'd like a more visible behavior with such a coding issue. Please replace this with a return statement (so no video), or any other idea you might have